### PR TITLE
Merge release/1.2.0 to master

### PR DESCRIPTION
**Pull request by rpdome**

### CODE REVIEW

File: changelog [hunk 1]
```diff
@@ -1,11 +1,19 @@
 MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki
 
+Version 1.2.0
+------------
+- Fixed issue #850, #890.
```
Comment (kreedula) [inline changelog]: I think we would need to add about the public api change here

File: msal/build.gradle [hunk 2]
```diff
@@ -144,9 +144,9 @@ dependencies {
         transitive = false
     }
 
-    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '1.0.13-hf1', changing: true)
+    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '1.0.14', changing: true)
```
Comment (shahzaibj) [inline msal/build.gradle]: I think we need `-SNAPSHOT` here

**Comment by shahzaibj** [comment]
> Left a comment

NVM, this is merge to master. Approved!

**Comment by rpdome** [comment]
> > Left a comment
> 
> NVM, this is merge to master. Approved!

I'll fix it anyway. Thanks for pointing this out!

**Pull request merged**